src/db.cpp

```diff
@@ -88,14 +85,14 @@ bool CDBEnv::Open(const boost::filesystem::path& path)
     dbenv.set_flags(DB_TXN_WRITE_NOSYNC, 1);
     dbenv.log_set_config(DB_LOG_AUTO_REMOVE, 1);
     int ret = dbenv.open(path.string().c_str(),
-                     DB_CREATE     |
-                     DB_INIT_LOCK  |
-                     DB_INIT_LOG   |
-                     DB_INIT_MPOOL |
-                     DB_INIT_TXN   |
-                     DB_THREAD     |
-                     DB_RECOVER,
-                     S_IRUSR | S_IWUSR);
+        DB_CREATE |
```

Comment:
might have to fix this manually but I actually would prefer if all of these had the splitting pipe at the same character position.